app/templates/_instructions.html

```diff
@@ -0,0 +1,174 @@
+    <span class="p">}</span>
+  <span class="p">}))</span>
+
+  <span class="c1">// see the data package object</span>
+  <span class="nx">console</span><span class="p">.</span><span class="nx">dir</span><span class="p">(</span><span class="nx">dp</span><span class="p">)</span>
+
+  <span class="c1">// data itself is stored in Resource object, e.g. to access first resource:</span>
+  <span class="nx">console</span><span class="p">.</span><span class="nx">log</span><span class="p">(</span><span class="nx">dp</span><span class="p">.</span><span class="nx">resources</span><span class="p">[</span><span class="mi">0</span><span class="p">].</span><span class="nx">_values</span><span class="p">)</span>
+
+  <span class="k">return</span> <span class="nx">dp</span>
+<span class="p">}</span>
+
+
+<span class="nx">fetchDataPackageAndData</span><span class="p">(</span><span class="s1">'https://bits.datapackaged.com/metadata/{{ dataset.owner }}/{{ dataset.name }}/_v/latest/datapackage.json'</span><span class="p">);</span>
+</pre>
+</div>
+
+<p>Our JavaScript is written using ES6 features. We are using <code>node.js v7.4.0</code> and passing <code>--harmony</code> option to enable ES6:</p>
+<p>
+<div class="highlight">
+<pre>$ node --harmony index.js</pre>
+</div>
+</p>
+{%- endmacro %}
+
+{% macro sql(dataset) -%}
+<p>In order to work with Data Packages in SQL you need to install our packages:</p>
+<div class="highlight"><pre><span></span>$ pip install datapackage
+$ pip install jsontableschema-sql
+$ pip install sqlalchemy
+</pre></div>
+
+<p>To import Data Package to your SQLite Database, run following code:</p>
+
+<div class="highlight">
+<pre>
+<span></span><span class="kn">import</span> <span class="nn">datapackage</span>
+<span class="kn">from</span> <span class="nn">sqlalchemy</span> <span class="kn">import</span> <span class="n">create_engine</span>
+
+<span class="n">data_url</span> <span class="o">=</span> <span class="s1">'https://bits.datapackaged.com/metadata/{{ dataset.owner }}/{{ dataset.name }}/_v/latest/datapackage.json'</span>
+<span class="n">engine</span> <span class="o">=</span> <span class="n">create_engine</span><span class="p">(</span><span class="s1">'sqlite:///:memory:'</span><span class="p">)</span>
+<span class="c1"># to load Data Package into storage</span>
+<span class="n">storage</span> <span class="o">=</span> <span class="n">datapackage</span><span class="o">.</span><span class="n">push_datapackage</span><span class="p">(</span><span class="n">data_url</span><span class="p">,</span> <span class="s1">'sql'</span><span class="p">,</span> <span class="n">engine</span><span class="o">=</span><span class="n">engine</span><span class="p">)</span>
+<span class="c1"># to see datasets in this package</span>
```

Comment:
Again let's stay consistent - there is no space between comment and previous code in Pandas and sql  instruction, but there is spaces in pyhon, js and R code snippets. Choose one if them. (think spaced one is better) Eg: 
This one is better
![image](https://cloud.githubusercontent.com/assets/15143785/25887058/89c78a8a-3571-11e7-8e9c-9af31e7c28e6.png)
then this 
![image](https://cloud.githubusercontent.com/assets/15143785/25887089/b03de6fa-3571-11e7-8d03-6d36648ac5e1.png)

Same with sql